Values doesn't implement Debug

<!--
Please use the following template to assist with creating an issue and to ensure a speedy resolution. If an area is not applicable, feel free to delete the area or mark with `N/A`
-->

### Rust Version

* rustc 1.26.1 (827013a31 2018-05-25)

### Affected Version of clap

* clap 2.31.2

### Expected Behavior Summary
Debug printing of `clap::Values` should be allowed.


### Actual Behavior Summary
Compilation error:
```
error[E0277]: `clap::Values<'_>` doesn't implement `std::fmt::Debug`
```

--- a/src/args/arg_matches.rs
+++ b/src/args/arg_matches.rs
@@ -786,8 +786,7 @@ impl<'a> ArgMatches<'a> {
 /// assert_eq!(values.next(), None);
 /// ```
 /// [`ArgMatches::values_of`]: ./struct.ArgMatches.html#method.values_of
-#[derive(Clone)]
-#[allow(missing_debug_implementations)]
+#[derive(Debug, Clone)]
 pub struct Values<'a> {
     iter: Map<Iter<'a, OsString>, fn(&'a OsString) -> &'a str>,
 }
@@ -838,8 +837,7 @@ impl<'a> Default for Values<'a> {
 /// ```
 /// [`ArgMatches::values_of_os`]: ./struct.ArgMatches.html#method.values_of_os
 /// [`Values`]: ./struct.Values.html
-#[derive(Clone)]
-#[allow(missing_debug_implementations)]
+#[derive(Debug, Clone)]
 pub struct OsValues<'a> {
     iter: Map<Iter<'a, OsString>, fn(&'a OsString) -> &'a OsStr>,
 }
@@ -890,8 +888,7 @@ impl<'a> Default for OsValues<'a> {
 /// assert_eq!(indices.next(), None);
 /// ```
 /// [`ArgMatches::indices_of`]: ./struct.ArgMatches.html#method.indices_of
-#[derive(Clone)]
-#[allow(missing_debug_implementations)]
+#[derive(Debug, Clone)]
 pub struct Indices<'a> { // would rather use '_, but: https://github.com/rust-lang/rust/issues/48469
     iter: Map<Iter<'a, usize>, fn(&'a usize) -> usize>,
 }

